mapper_test.go

```diff
@@ -128,6 +128,60 @@ func TestMetricMapper(t *testing.T) {
 			`,
 			configBad: true,
 		},
+		// Config with only comments
```

Comment:
I'm glad to make the change. Kindly elaborate on what you mean by "end with a full stop."